sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormTrainingClient/FormTrainingClientLiveTests.cs

```diff
@@ -133,9 +133,11 @@ public async Task CheckFormTypeinSubmodelAndRecognizedForm(bool labeled)
         }
 
         [Test]
-        public async Task StartCreateComposedModel()
+        [TestCase(false)]
+        [TestCase(true)]
+        public async Task StartCreateComposedModel(bool useTokenCredential)
```

Comment:
should this be called StartCreateComposedModelCanAuthenticateWithTokenCredential()

Reply:
If the value of `useTokenCredential` is false, then this test will run with Api key credentials. I reused this tests to avoid duplicated code.
Apart from this PR, I am looking into a way to make this true in our TestFramework, so I don't have to manually add a parameter, but instead let the framework do it for me.